view/frontend/templates/script.phtml

```diff
@@ -1,4 +1,4 @@
 <?php /** @var Magento\Framework\View\Element\Template $block */ ?>
 <?php /** @var CustomGento\Cookiebot\ViewModel\Script $viewModel */ ?>
 <?php $viewModel = $block->getData('view_model') ?>
-<?= $viewModel->getScript() ?>
+<?= $viewModel->getScript() // phpcs:ignore ?>
```

Comment:
Why ignore? This should get a `/* @noEscape */`.

Reply:
Since the output is static and does not need to be escaped